### PR TITLE
Test add some debug to see why the CI URL fetch is failing

### PR DESCRIPTION
**Pull request by chmouel**
/hold

### CODE REVIEW

**Comment by piyush-garg** [comment]
@chmouel This script is executed on Jenkins, not as part of openshift CI, I think you will see no change here.

**Comment by chmouel** [comment]
@piyush-garg why is it launched on jenkins ? I am not sure i have followed this,

**Comment by piyush-garg** [comment]
It is run as part of update-to-head.sh script as we need the release.yaml in the codebase of operator

**Comment by chmouel** [comment]
Oh I get it now, that's a very convulsed setup to be able to publish some artifacts at the end.....

so i guess i need to get this tested with jenkins.

**Comment by chmouel** [comment]
/hold cancel

let's merge at least we will get some debug

**Comment by chmouel** [comment]
let's remove getting it from nightly and merge so we can test the fallback mechanism asap, this will be removed in a next commit..

File: openshift/release/fetch-pipeline.sh [hunk 1]
```diff
@@ -8,14 +8,21 @@
...
 NIGHTLY_RELEASE="https://raw.githubusercontent.com/openshift/tektoncd-pipeline/release-next/openshift/release/tektoncd-pipeline-nightly.yaml"
 STABLE_RELEASE_URL='https://raw.githubusercontent.com/openshift/tektoncd-pipeline/${version}/openshift/release/tektoncd-pipeline-${version}.yaml'
 PAYLOAD_PIPELINE_VERSION="release-next"
 
 function get_version {
     local shift=${1} # 0 is latest, increase is the version before etc...
-    local version=$(python -c "from pkg_resources import parse_version;import sys, json, requests;jeez=requests.get('https://api.github.com/repos/tektoncd/pipeline/releases').json();print(sorted([x['tag_name'] for x in jeez], key=parse_version, reverse=True)[${shift}])")
+    tmpf=$(mktemp /tmp/.XXXXXX)
+    curl -f ${CURL_OPTIONS} -o ${tmpf} -v https://api.github.com/repos/tektoncd/pipeline/releases
+    local version=$(python -c "from pkg_resources import parse_version;import sys, json;jeez=json.load(open('${tmpf}'));print(sorted([x['tag_name'] for x in jeez], key=parse_version, reverse=True)[${shift}])")
```
Comment (piyush-garg) [inline openshift/release/fetch-pipeline.sh:25]: No need to import `sys` now

File: openshift/release/fetch-pipeline.sh [hunk 2]
```diff
@@ -8,14 +8,21 @@
...
+
 MAX_SHIFT=1
 NIGHTLY_RELEASE="https://raw.githubusercontent.com/openshift/tektoncd-pipeline/release-next/openshift/release/tektoncd-pipeline-nightly.yaml"
 STABLE_RELEASE_URL='https://raw.githubusercontent.com/openshift/tektoncd-pipeline/${version}/openshift/release/tektoncd-pipeline-${version}.yaml'
 PAYLOAD_PIPELINE_VERSION="release-next"
 
 function get_version {
     local shift=${1} # 0 is latest, increase is the version before etc...
-    local version=$(python -c "from pkg_resources import parse_version;import sys, json, requests;jeez=requests.get('https://api.github.com/repos/tektoncd/pipeline/releases').json();print(sorted([x['tag_name'] for x in jeez], key=parse_version, reverse=True)[${shift}])")
+    tmpf=$(mktemp /tmp/.XXXXXX)
```
Comment (piyush-garg) [inline openshift/release/fetch-pipeline.sh:23]: we can remove this later

File: openshift/release/fetch-pipeline.sh (hunk 1 above)
Comment (piyush-garg) [inline openshift/release/fetch-pipeline.sh:25]: Will single quotes in open() works?

Comment (chmouel) [reply openshift/release/fetch-pipeline.sh:25]: yes it does!

**Pull request merged**